lib/ex_twilio/request_validator.ex

```diff
@@ -28,13 +26,12 @@ defmodule ExTwilio.RequestValidator do
     params
     |> Map.keys()
     |> Enum.sort()
-    |> Enum.map(fn key -> key <> Map.get(params, key) end)
-    |> Enum.join()
+    |> Enum.map_join(fn key -> key <> Map.get(params, key) end)
   end
 
   defp compute_hmac(data, key), do: hmac(:sha, key, data)
 
-  # TODO: remove when we require OTP 22
+  # remove when we require OTP 22
```

Comment:
I wonder if it's really worth having that TODO check if we still leave notes like this in the codebase? if anything it's maybe less easy to find this comment now that there's no TODO?

i dont know if i really care one way or another, but just kinda seems like a useless check?

Reply:
yeah i think that makes sense, i don't really see any issue with TODO comments (as long as a ticket is created to correspond to it i guess)